liam2/simulation.py

```diff
@@ -241,6 +242,12 @@ def from_str(cls, yaml_str, simulation_dir='',
         #                'MIG': {'type': int}}
         globals_def = {}
         for k, v in content.get('globals', {}).iteritems():
+            if 'value' in v:

```

Comment:
this whole block of change could be removed IMO because it is mostly unneeded. All it does is transform {'INT_CONSTANT': {'value': 42, 'type': 'int'}} to {'INT_CONSTANT': 42}

I would rather avoid doing that and simply handle the "value" case in globals_symbols. Besides, I prefer if from_str is does not change the structure of the globals definition, just convert string (types) to real types.